lib/discourse_ip_info.rb

```diff
@@ -25,17 +25,34 @@ def self.mmdb_path(name)
   end
 
   def self.mmdb_download(name)
+    extra_headers = {}
+
     url =
       if GlobalSetting.maxmind_mirror_url.present?
         File.join(GlobalSetting.maxmind_mirror_url, "#{name}.tar.gz").to_s
       else
-        if GlobalSetting.maxmind_license_key.blank?
-          STDERR.puts "MaxMind IP database updates require a license"
-          STDERR.puts "Please set DISCOURSE_MAXMIND_LICENSE_KEY to one you generated at https://www.maxmind.com"
+        license_key = GlobalSetting.maxmind_license_key
+
+        if license_key.blank?
+          STDERR.puts "MaxMind IP databases download require a acconunt ID and license key"
+          STDERR.puts "Please set DISCOURSE_MAXMIND_ACCOUNT_ID and DISCOURSE_MAXMIND_LICENSE_KEY. See https://meta.discourse.org/t/configure-maxmind-for-reverse-ip-lookups/173941 for more details."
```

Comment:
The meta guide doesn't mention the account ID, we will update it once this PR is merged?